storage/statedb/cache.go

```diff
@@ -16,39 +16,72 @@
 
 package statedb
 
-import "errors"
+import (
+	"errors"
+	"strings"
+)
+
+type TrieNodeCacheType string
+
+// TrieNodeCacheConfig contains configuration values of all TrieNodeCache.
+type TrieNodeCacheConfig struct {
+	CacheType          TrieNodeCacheType
+	FastCacheSizeMB    int      // Memory allowance (MB) to use for caching trie nodes in fast cache
+	RedisEndpoints     []string // Endpoints of redis cache
+	RedisClusterEnable bool     // Enable cluster-enabled mode of redis cache
+}
 
-type CacheType string
+// TrieNodeCache interface the cache of stateDB
+type TrieNodeCache interface {
+	Set(k, v []byte)
+	Get(k []byte) []byte
+	Has(k []byte) ([]byte, bool)
+}
 
 const (
-	// Available stateDB cache types
-	LocalCache  CacheType = "LocalCache"
-	RemoteCache           = "RemoteCache"
-	HybridCache           = "HybridCache"
+	// Available trie node cache types
+	CacheTypeFast   TrieNodeCacheType = "FastCache"
```

Comment:
I think `LocalCache` is better because several local cache types are not necessary.

Reply:
Renamed "FastCache" to "LocalCache" and the variable name `CacheTypeFast` to `CacheTypeLocal`.